### PR TITLE
General submodule update

### PR DESCRIPTION
**Pull request by GiudGiud**
, refs #0

### CODE REVIEW

**Comment by GiudGiud** [comment]
I think there s a deprecation problem in SAM
when i use Head instead of Head_fcn
https://civet.inl.gov/job/2101772/
```
data/civet1/build/virtual_test_bed/pbfhr/sam_model.transient_run: PBPump 'Pump' expects a non-zero constant user-specified pump head value as an initial guess when a mass flow rate is desired.
```

**Comment by GiudGiud** [comment]
@travismui 
anything i can do to avoid this deprecation?
https://civet.inl.gov/job/2106230/

**Comment by travismui** [comment]
> @travismui anything i can do to avoid this deprecation? https://civet.inl.gov/job/2106230/

Thanks @GiudGiud for the heads up and fixing the other issue. This one will take a little more effort to fix, but I'll take a look at it and push updates to this branch if allowed.

**Comment by travismui** [comment]
> > @travismui anything i can do to avoid this deprecation? https://civet.inl.gov/job/2106230/
> 
> Thanks @GiudGiud for the heads up and fixing the other issue. This one will take a little more effort to fix, but I'll take a look at it and push updates to this branch if allowed.

OK actually this deprecation warning should not be issued, it will be removed from SAM shortly. I can push a submodule update to this branch after we get it into SAM master.

File: lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i [hunk 1]
```diff
@@ -547,1341 +547,1341 @@ totalpower = 3700000.0 # W
...
-       CA46:1.394535E-09
-       CA48:6.519498E-08
-       NB93:7.519752E-06
-       N14:4.969370E-05
-       N15:1.835515E-07
-       AL27:2.589280E-05
-       TA181:3.861021E-06
-       B10:5.144462E-06
-       B11:2.070704E-05'
+       FE54 3.185952E-03
```
Comment (GiudGiud) [inline lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i:679]: @YaqiWang how do I fix this?
FE54 is present in two 'sub-compositions' in that map (it s not a thing maybe) so the code complains of a duplicate

**Comment by GiudGiud** [comment]
@bwspenc who is a good person to clear that this diff on a tensor mechanics is normal and due to changes in BISON( maybe?)
https://civet.inl.gov/job/2106235/

File: lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i (hunk 1 above)
Comment (YaqiWang) [reply lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i:679]: @GiudGiud each composition should be containing only one set of isotopes. The materials of the old input have multiple compositions, you will need to add multiple compositions for each material. I think this input was created by Hansol at ANL. You can ask some advices from him on griffin slack channel. I am not sure whether all materials have different extra element ids.

Comment (GiudGiud) [reply lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i:679]: @pak1sol do you have a script to merge these compositions together?

**Comment by bwspenc** [comment]
@GiudGiud did you still need help with the bison tests? Looks like they're passing now.

**Comment by GiudGiud** [comment]
I loosened the tolerance in prevision. It would still be good to check

**Comment by GiudGiud** [comment]
Getting failures on that griffin input. Merging with that failure, we ll have to fix it this week

**Pull request merged**